compute_endpoint/PyPI.md

```diff
@@ -0,0 +1,7 @@
+# Globus Compute Endpoint
+
+[Globus Compute](https://globus-compute.readthedocs.io/en/latest/) is a distributed Function as a Service (FaaS) platform that enables flexible, scalable, and high performance remote function execution. Unlike centralized FaaS platforms, Globus Compute allows users to execute functions on heterogeneous remote computers, from laptops to campus clusters, clouds, and supercomputers.
+
+This package provides the functionality for running and managing [endpoints](https://globus-compute.readthedocs.io/en/latest/endpoints.html), which are the services that manage the execution of functions on target machines.
```

Comment:
This paragraph make endpoint sound like web-service.  ie. manage the execution of functions, not actually running alongside the workers on the end node.  Maybe make it more clear?

Reply:
A friendly callout of the naked &lsquo;endpoints&rsquo;:

```diff
- ... and managing [endpoints](...
+ ... and managing [compute endpoints](...
```

Reply:
Perhaps:
> The package provides the [Compute Endpoint](https://globus-compute.readthedocs.io/en/latest/endpoints.html) agent&nbsp;&mdash;&nbsp;the software that receives tasks (functions + arguments) from the web-services and manages execution on target machines.